src/modules/launcher/Wox.Core/Plugin/PluginManager.cs

```diff
@@ -201,7 +216,8 @@ private static List<Result> UpdateResultWithActionKeyword(List<Result> results,
 
                 if (!string.IsNullOrEmpty(query.ActionKeyword))
                 {
-                    result.QueryTextDisplay = string.Format("{0} {1}", query.ActionKeyword, result.QueryTextDisplay);
+                    // Using InvariantCulture since this is a command line arg
+                    result.QueryTextDisplay = string.Format(CultureInfo.InvariantCulture, "{0} {1}", query.ActionKeyword, result.QueryTextDisplay);
```

Comment:
This is a user facing string so shouldn't this be `Currentculture` instead of `InvariantCulture`?

Reply:
Thanks @alekhyareddy28. Looking at the available options, would this be `CurrentCulture `or `CurrentUICulture`?

Reply:
You should get a compile error if you use the wrong one.